src/css/controls/flags/overlay.less

```diff
@@ -17,7 +13,7 @@
 
     &-sharing:not(.jw-flag-small-player) {
         .jw-controls,
-        .jw-title {
+        .jw-controls-right .jw-logo {
```

Comment:
This is the exception where the title should be shown, so let's keep `.jw-title` here